config/routes.rb

```diff
@@ -33,12 +31,14 @@
       BODY
     end
 
-  specified_get '/ref/:reference/:repository_id/*locid' => 'ontologies#show',
+  specified_get '/ref/:reference/:repository_id/*locid' => 'api/v1/ontology_versions#show',
```

Comment:
You could try moving routes.rb to the excluded files-list. Since the file is written in a DSL i would argue that it doesn't need to adhere to the same standards as *normal* code.